@stellar/freighter-api/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@stellar/freighter-api",
-  "version": "1.4.1",
```

Comment:
Do we want to keep this version on the current one?

Reply:
Oh yup, noticed this, too. This was actually a mistake to begin with. The current version on NPM is 1.4.0